scripts/install_third_party_libs.py

```diff
@@ -69,6 +69,25 @@
     common.OPPIA_TOOLS_DIR, 'pylint-quotes-0.1.8', 'configparser.py')
 
 
+def tweak_yarn_executable():
+    """When run yarn on Windows, the file `yarn` will be excecuted (
+    even then command specified is `yarn.cmd`). While this file is a bash
+    script, so it can not be executed. To avoid the issue, we can rename
```

Comment:
How does renaming fix the issue? Won't Windows still search for `yarn`?

Reply:
On windows, the command would be `yarn.cmd`. When there is no `yarn` file in the path, it will try to find the `yarn.cmd` file.

Reply:
So if I understand correctly, we're renaming yarn to yarn.sh to avoid it being detected by Windows as the default executable, and so that yarn.cmd is what gets detected instead. Is that right?

If so -- why don't we just delete the yarn file?

(Sorry if I am misunderstanding.)

Reply:
In case someone may want to use the `yarn.sh` file.

Although the file can not be executed directly on Windows's default shells like cmd and PowerShell, it can be executed via git bash. So if someone is using bash they can still run the `yarn.sh` via `yarn` command.